packages/math/src/bn.test.ts

```diff
@@ -280,4 +280,152 @@ describe('Math - Convert', () => {
     expect(bn(maxBytes).toHex(4)).toEqual(maxHex);
     expect(() => bn(over).toHex(4)).toThrow();
   });
+
+  it('should accept possibly undefined to craete bn, and set to 0 when undefined', () => {
```

Comment:
```suggestion
  it('should create bn with number or undefined', () => {
```

Reply:
maybe add some tests for creating with string also?

Reply:
this test is only check undefined compatibility. we have a lot of other tests creating from strings already